libs/ui-components/src/components/Catalog/AddCatalogItemWizard/utils.ts

```
@@ -0,0 +1,513 @@
+        return semver.valid(value) !== null;
+      })
+      .test('unique-version', t('Version must be unique'), (value) => {
+        if (!value) {
+          return true;
+        }
+        return !duplicates.has(value);
+      }),
+    refType: Yup.string().oneOf(['tag', 'digest']).required(),
+    tag: Yup.string().when('refType', {
+      is: 'tag',
+      then: (schema) => schema.required(t('Tag is required')),
```

Biome (2.4.6)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)